api/src/modules/indicator-records/indicator-records.service.ts

```diff
@@ -149,7 +149,7 @@ export class IndicatorRecordsService extends AppBaseService<
                 sourcingRecord.id,
               );
             break;
-          case 'GHG_LUC_T':
+          case 'carbon_emissions':
```

Comment:
Even tho now is way more readable, we are still using magic strings here. 

Please have a look to h3-data.service.ts, function `getRiskMapByResolution` to check how we are avoiding using said magic strings there to get an idea